djangoproject/templates/aggregator/index.html

```diff
@@ -65,26 +66,30 @@ <h2 class="bullet-icon"><i class="icon icon-rss blue"></i> {{ feedtype.name }}</
           <dl class="list-links">
             {% for item in latest_feeds %}
               <dt><a href="{{ item.link }}">{{ item.title }}</a></dt>
-              <dd>{{ item.date_modified|date:"N jS, Y \a\t P" }} by <a href="{{ item.feed.public_url }}">{{ item.feed.title }}</a></dd>
+              <dd>
+                {% blocktranslate trimmed with modified_date=item.date_modified|date:"N jS, Y \a\t P" public_url=item.feed.public_url title=item.feed.title %}
+                  {{ modified_date }} by <a href="{{ public_url }}">{{ title }}</a>
+                {% endblocktranslate %}
+              </dd>
             {% endfor %}
           </dl>
           <p class="meta">
             {% if latest_feeds %}
-              <a href="{% url 'community-feed-list' feedtype.slug %}">View more</a>
+              <a href="{% url 'community-feed-list' feedtype.slug %}">{% translate "View more" %}</a>
             {% endif %}
             {% if latest_feeds and feedtype.can_self_add %}
-              or
+              {% translate "or" %}
```

Comment:
This kind of piecemeal translation might not very well for some languages I can't come up with a concrete example, but I could imagine a language where the word "or" might need to be translated differently based on context.

A solution might be a refactor into a less clever if/elif/else structure where full sentences are translated.
If that's too complicated, then at the minimum we should probably use a [contextual marker](https://docs.djangoproject.com/en/5.1/topics/i18n/translation/#contextual-markers):
```suggestion
              {% translate "or" context "view more OR add your feed" %}
```